ci-operator/step-registry/assisted/verify-generated-code/assisted-verify-generated-code-ref.yaml

```diff
@@ -5,9 +5,9 @@ ref:
   commands: assisted-verify-generated-code-commands.sh
   resources:
     requests:
-      cpu: "3"
-      memory: 600Mi
+      cpu: "4"
+      memory: 1650Mi
```

Comment:
This is the same as the limit.

Reply:
Is that bad?

Reply:
I thought that was the ideal (Guaranteed QoS class etc)